e2e/bittwister/suite_setup_test.go

```diff
@@ -0,0 +1,41 @@
+package bittwister
+
+import (
+	"context"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/celestiaorg/knuu/pkg/knuu"
+)
+
+type Suite struct {
+	suite.Suite
+	Knuu *knuu.Knuu
+}
+
+func (s *Suite) SetupSuite() {
+	var (
+		err error
+		ctx = context.Background()
+	)
+	s.Knuu, err = knuu.New(ctx, knuu.WithProxyEnabled())
+	s.Require().NoError(err)
+	s.T().Logf("Scope: %s", s.Knuu.Scope())
+	s.Knuu.HandleStopSignal(ctx)
+}
+
+func (s *Suite) TearDownSuite() {
+	s.T().Cleanup(func() {
```

Comment:
Not sure why the Teardown function was called prematurely, so I had to add it to the test cleanup func so it is called once all tests are done.